package.json

```diff
@@ -27,13 +27,14 @@
     "format": "prettier --write '**/*.{ts,js,json,css,md,yml}'",
     "type-check": "tsc --noEmit",
     "test": "jest",
-    "test:ci": "yarn lint && yarn type-check && yarn test",
+    "test:ci": "yarn lint && yarn type-check && yarn test -- --ci",
```

Comment:
I don't think the double dashes are necessary with yarn. (I know it is for `npm run` commands though)

Reply:
I'm not sure why that's still there I already removed them

Reply:
ahh k I see that now